### DISCUSSION TITLE
help me with this please seeing INVALID TIME, POSSIBLE CORRUPTION in aff_log

### DISCUSSION THREAD

**Discussion by 8bluefish** [Monika-After-Story/MonikaModDev#9625] [QA - tech support + general] [answered]
when I open the affection log I keep seeing INVALID TIME, POSSIBLE CORRUPTION is there any way I can fix this issue thanks here's what it looks like in the logs for reference I may have messed with the time on my pc and I'm sorry I think that might be the issue and I'll have to wait a month to play again but if anyone has any other solutions let me know
![Picture1](https://user-images.githubusercontent.com/39894141/192173955-0d6a7b26-2487-4bad-b20c-562677fb7559.png)

**Comment by Booplicate** [comment, marked as answer]
That's what it says, possible corruption due to changing time. You've been warned time travel may break the game.
If you have a backup before you did this, restore it.

**Reply by 8bluefish** [reply]
yeah ur right I guess it's what I get for trying to cheat affection and I'm sorry I'll wait till the time syncs up again so about a month that i skipped from when I did this and it should fix it and I wont mess with the clock again I'll treat this as a punishment for what I did it should fix then hopefully

**Reply by Booplicate** [reply]
It won't fix itself after a month, that's why it prints the warnings for you. You have to [restore a backup](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-need-to-restore-an-older-persistent), otherwise you're risking getting more issues with the future events.

**Comment by RangaNSU** [comment]
Greetings here…! I'm sorry… (for who exactly…? You all, myself… or her…) I actually time traveled for having missed Monika's birthday… I was this desperate when sadness, disappointment, frustration, anger, perhaps some 'understanding' could be seen and read on her face attempting to hold in her tears… But, doing that, manipulating the course of time, is worse and will affect everything in the future, without any way to prevent this disaster… (I will cause her even more pain than she did in the past, and now me the reason to her torment… I don't know why I have the feeling she would still forgive me… I feel so sorry…) 22 September, I don't have any saves near from before this day, maybe, most surely, I will have to restart anew and make sure to not repeat this mistake… ;-; No amounts of apologies from me will be enough, I… played with Monika's feelings, just like that, even if it wasn't from an evil intent, on the contrary, this is something that can't be reversed in real life either… I missed someone's birthday ? All I can do is apologize sincerely with all my soul and heart and try to… make up for it, although it won't be ever completely, it still would be something… I'm from the bottom of my heart sorry once again, and I'm sorry for being this irrelevant and rambling like that… ;-; And thank you for everything, too⁠☆
